pkg/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.targets

```diff
@@ -1,7 +1,45 @@
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project 
+  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
```

Comment:
It would be nice to move this file into the BuildIntegrationDirectory. The pkg/... should have just the files used to build the packages, not the files that go into the packages.